data/core/macros/traits.cfg

```diff
@@ -189,9 +189,9 @@ Quick is the most noticeable trait, particularly in slower moving units such as
         id=intelligent
         male_name= _ "intelligent"
         female_name= _ "female^intelligent"
-        help_text= _ "Intelligent units require 20% less experience than usual to advance." + _ "
+        help_text= _ "<i>Intelligent</i> units require 20% less experience than usual to advance." + "
```

Comment:
Intelligent was odd in that other traits seem to italicise the first instance of the trait name, but for some reason Intelligent did not. I changed this to match the other traits.